src/absenceDays/types.ts

```diff
@@ -1,16 +1,16 @@
 export type AbsenceDay = {
-  id?: number;
```

Comment:
All mandatory?

Reply:
Yes, because absenceDays has only a read endpoint and we always have all fields.

And for unit testing we need types with different mandatory fields.